ptvsd/attach_server.py

```diff
@@ -90,11 +81,9 @@ def break_into_debugger():
     if not _attached.isSet() or debugger is None:
         return
 
-    thread = pydevd.threadingCurrentThread()
-    try:
-        additional_info = thread.additional_info
-    except AttributeError:
-        additional_info = PyDBAdditionalThreadInfo()
-        thread.additional_info = additional_info
-
-    debugger.set_suspend(thread, CMD_THREAD_SUSPEND)
+    import sys
+    pydevd.settrace(
+        suspend=True,
+        trace_only_current_thread=True,
```

Comment:
Why is `trace_only_current_thread=True`
Why can't and shouldn't we be tracing other threads?

Reply:
`break_into_debugger` will only make the pause for the current thread. The method `enable_attach` is the one responsible for setting the tracing.